frameworks/react/container/package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "react-container-starter",
-  "version": "17.0.0",
+  "version": "19.0.0",
   "private": true,
   "dependencies": {
     "@finos/fdc3": "2.0.3",
-    "@openfin/core": "34.78.80",
-    "@openfin/workspace": "17.0.15",
-    "@openfin/node-adapter": "34.78.80",
-    "@testing-library/jest-dom": "5.17.0",
-    "@testing-library/react": "13.4.0",
-    "@testing-library/user-event": "13.5.0",
-    "@types/jest": "27.5.2",
-    "@types/node": "16.18.60",
-    "@types/react": "18.2.35",
-    "@types/react-dom": "18.2.14",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-router-dom": "6.18.0",
+    "@openfin/core": "38.82.69",
```

Comment:
Could we match the runtime used by workspace v19? So that if you grab a branch you know all the examples are running on the runtime of a particular release (container can use any release but it just makes it easy to be more consistent across branches).

Reply:
Changed to 38.82.64